projects/rawkode.academy/platform/web/src/components/blog/ArticleCard.astro

```diff
@@ -6,14 +6,20 @@ type Props = {
 };
 
 const { article } = Astro.props;
+
+let link = `/blog/${article.id}`;
+
+if (article.data.series) {
+	link = `/blog/series/${article.id}`;
```

Comment:
The only reasons I did this was:
* an article with no series has the slug `/blog/<article-slug>` and the breadcrumb `Blog > Article name`, which is okay
* an article with a series has the slug `/blog/<article-slug>` but the breadcrumb `Blog > Series > Name of the series > Article name`, which is super weird

I don't think we should get rid of the breadcrumb, because it helps navigating the content and the URI should behave accordingly. I hope this makes sense.